internal/guest/runtime/hcsv2/container.go

```diff
@@ -66,6 +66,7 @@ func (c *Container) ExecProcess(ctx context.Context, process *oci.Process, conSe
 		return -1, err
 	}
 
+	process.Rlimits = c.spec.Process.Rlimits
```

Comment:
are we guaranteed that there's no nil deref?

Reply:
Let me check..

Reply:
I don't believe it's possible at this point to have an empty spec (or process in the spec) as the container would be created at this point. I could add a check regardless though to avoid any potential future problems

Reply:
I think it's fine then. thanks for checking.